sandcastle/utils.py

```diff
@@ -97,11 +97,14 @@ def clean_string(s: str) -> str:
     '[a-z0-9]( [-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'
 
     'must be no more than 63 characters'
+
+    remove 21 characters for make room to the timestamp
+    remove 5 characters for make room to the - {{ deployment }} -
```

Comment:
Could you please clarify why is this needed?

Reply:
I rewrote it in a different way, let me know if now it is clear.

Reply:
That's better, thanks, but it still isn't clear to me why there is now a need to make a room for a timestamp (that will be appended to the pod name I assume) and it wasn't before.

Reply:
I mean, was it a bug?